include/onemkl/blas/blas.hpp

```diff
@@ -1814,6 +1680,1928 @@ static inline void trsv(cl::sycl::queue &queue, uplo upper_lower, transpose tran
     trsv_postcondition(queue, upper_lower, trans, unit_diag, n, a, lda, x, incx);
 }
 
+// USM APIs
+
+static inline cl::sycl::event asum(cl::sycl::queue &queue, std::int64_t n,
+                                   const std::complex<float> *x, std::int64_t incx, float *result,
+                                   const cl::sycl::vector_class<cl::sycl::event> &dependencies) {
```

Comment:
`dependencies` should default to an empty vector -- `dependencies = {}`

Reply:
Done